lists/tr.csv

```diff
@@ -178,3 +178,20 @@ https://www.bbc.com/turkce,NEWS,News Media,2018-09-13,OONI,
 https://www.dw.com/tr/,NEWS,News Media,2018-09-13,OONI,
 https://tr.sputniknews.com/,NEWS,News Media,2018-09-13,OONI,
 https://www.amerikaninsesi.com/,NEWS,News Media,2018-09-13,OONI,
+http://etha10.com/,NEWS,News Media,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+https://ozgurgelecek17.net/,NEWS,News Media,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+http://osp.org.tr/,POLR,Political Criticism,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+http://geziyisavunuyoruz.org/,POLR,Political Criticism,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+https://www.gazetefersude.com/,NEWS,News Media,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+https://yenidemokratgenclik1.com/,POLR,Political Criticism,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+http://umutgazetesi14.org/,NEWS,News Media,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
+http://kizilbayrak44.net/,NEWS,News Media,2019-08-06,,Ankara 3rd Criminal Judgeship of Peace court order 2019/5538
```

Comment:
This site redirects to https when visiting it. We should be sure the HTTPS version of this (and other sites included in this PR that support HTTPS) are used for the list.

Reply:
I simpy copied the URLs from the court order ;) I will check all URLs and change them if they support HTTPS.